### ISSUE TITLE
Impossible to install on Windows

### ISSUE THREAD

**Issue by JacobCZ** [Block8/PHPCI#698]
Hey, so I've jsut decided to install PHPCI on my Windows machine and I have found this little problem, that prevents PHPCI from installing. Every time I run the `php ./console phpci:install` command, I get all the way to the `Setting up your database...` and then, this error appears: `'C:\Program' is not recognized as an internal or external command, operable program or batch file` and the installation continues, but then this appears: `SQLSTATE[42S02]: Base table or view not found: 1146 Table 'phpci.user' doesn't exist` and the installation breaks...

So what could be the problem?

**Comment by mavimo** [comment]
@JacobCZ do you try to change from:

    shell_exec(PHPCI_DIR . 'vendor/bin/phinx migrate -c "' . PHPCI_DIR . 'phinx.php"');

to:

    shell_exec('"' . PHPCI_DIR . 'vendor/bin/phinx" migrate -c "' . PHPCI_DIR . 'phinx.php"');

and test installation on Windows, it work as expected?

**Comment by AustP** [comment]
@mavimo I ran into the issue of running `exec` on Windows with a space in the path. I wasn't installing PHPCI but I can tell you what I found. Adding quotes did not work. Adding double quotes did not work. Using `escapeshellarg` did not work.  Finally I stumbled upon http://stackoverflow.com/questions/4226164/weird-bug-in-php-spaces-in-paths-and-windows

I was able to get my project working by doing what the selected answer there says. `cd` into the directory containing the executable and then execute it. Hope this helps.
